### PR TITLE
feat: booking notifications

### PR DESCRIPTION
**Pull request by momentiris**
Adds notifications for when a booking changes status.

Some any types introduced because I think the types throughout the application are out of scope here..

### CODE REVIEW

File: packages/engine-server/lib/helpers.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    status: booking.status,
+  },
+  booking,
+})
+
+const transportToNotification = (transport) => ({
+  /* We currently
+   * only have notifications for when a transport
+   * has been created, hence the 'success' severity status
+   * and the 'new' event.event.
```
Comment (momentiris) [inline packages/engine-server/lib/helpers.js:28]:
```suggestion
   * and the 'new' event.status.
```

File: packages/engine-ui/src/components/Notification.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import Zoom from '@material-ui/core/Zoom'
+import { Notification } from '../notificationTypes'
+import styled from 'styled-components'
+
+const StyledAlert = styled(Alert)`
+  align-items: center;
+`
+
+const Component: React.FC<
+  { notification: Notification } & { handleOnClose: (id: string) => void }
```
Comment (momentiris) [inline packages/engine-ui/src/components/Notification.tsx:12]:
```suggestion
  { notification: Notification, handleOnClose: (id: string) => void }
```

File: packages/engine-ui/src/components/Notifications.tsx [hunk 3]
```diff
@@ -1,130 +1,109 @@
...
-  </Zoom>
-)
+const notificationMessages = {
+  [EntityType.BOOKING]: {
+    [BookingStatus.NEW]: 'En ny bokning har lags till:',
+    [BookingStatus.ASSIGNED]: 'En bokning har tilldelats transport:',
+    [BookingStatus.DELIVERED]: 'En bokning har levererats:',
+    [BookingStatus.PICKED_UP]:
+      'En bokning har hämtats upp av sin tilldelade transport:',
+    [BookingStatus.DELIVERY_FAILED]: 'Bokning  kunde inte levereras:',
```
Comment (momentiris) [inline packages/engine-ui/src/components/Notifications.tsx:36]:
```suggestion
    [BookingStatus.DELIVERY_FAILED]: 'En bokning kunde inte levereras:',
```

File: packages/engine-ui/src/utils/helpers.ts [hunk 4]
```diff
@@ -40,10 +40,13 @@ const calculateMinTime = (date?: Date, minDate?: Date) => {
   return moment().startOf('day').toDate() // set to 12:00 am today
 }
 
-const getLastFourChars = (str: string) => str.slice(str.length - 4, str.length)
+const getLastFourChars = (str: string): string =>
```
Comment (momentiris) [inline packages/engine-ui/src/utils/helpers.ts:43]:
```suggestion
const getLastFourChars = (str: string) =>
```

**Review by lundeeell — APPROVED** [review]
Nice! Just remove accepted from statuses as it just spams :)

**Review by lundeeell — CHANGES_REQUESTED** [review]
duplicate queue name

**Review by lundeeell — APPROVED** [review]
👍

**Pull request merged**